test/runtests.jl

```diff
@@ -111,5 +111,53 @@ using Test
             @test call1 != call3
             @test call2 != call4
         end
+
+        @testset "splatting" begin
+            f(a, b; kw1=1, kw2=2) = a + b + kw1 + kw2
+
+            one = 1
+            a = [1, 2]
+            kw = (kw1=1, kw2=2)
+
+            @testset "basic splatting" begin
+                @test f(a...) == @cached_call f(a...)
+                @test f(a...) == @cached_call f([1, 2]...)
+                @test f(a...) == @cached_call f([one, 2]...)
+                @test f(a...; kw...) == @cached_call f(a...; kw...)
+                @test f(a...; kw...) == @cached_call f(a...; (kw1=1, kw2=2)...)
+                @test f(a...; kw...) == @cached_call f(a...; (kw1=one, kw2=2)...)
```

Comment:
how is `one` interpreted?

Reply:
It should be interpreted as its literal value, `1`.